applications/GeoMechanicsApplication/custom_constitutive/linear_elastic_law.cpp

```diff
@@ -102,11 +100,8 @@ Vector& GeoLinearElasticLaw::CalculateValue(ConstitutiveLaw::Parameters& rParame
                                             const Variable<Vector>&      rThisVariable,
                                             Vector&                      rValue)
 {
-    if (rThisVariable == STRAIN || rThisVariable == GREEN_LAGRANGE_STRAIN_VECTOR ||
```

Comment:
I removed this branch, however we could also throw an error, such that we're absolutely sure we know when this happens